### ISSUE TITLE
A question about detection rate

### ISSUE THREAD

**Issue by kisstea** [FaceDetect/jointCascade_py#3]
I find the code removes positive samples after each CART as the paper writes. My question is that, even I set a high precision-recall such as 0.999. After 5 stages, the positive remained is 0.999^(5040)=0.006457.  I wonder if the detection rate is too poor?

**Comment by samuel1208** [comment]
Yes, your are right,   The paper didn't  mention this either.  I advice you  set the recall with 1 in some stages,  If have any good strategies. please write down here

**Comment by samuel1208** [comment]
I think you can try the soft cascade method to set recall
